Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1487,10 +1489,13 @@ private extension Purchases {
     @objc func applicationWillEnterForeground() {
         Logger.debug(Strings.configure.application_foregrounded)
 
+        guard !self.shouldSkipCacheUpdate() else { return }
+
         // Note: it's important that we observe "will enter foreground" instead of
         // "did become active" so that we don't trigger cache updates in the middle
         // of purchases due to pop-ups stealing focus from the app.
         self.updateAllCachesIfNeeded(isAppBackgrounded: false)
+        self.setLastCacheUpdateTime()
         self.dispatchSyncSubscriberAttributes()
```

Comment:
This is now throttled in the `applicationWillEnterForeground` but not on the `applicationDidEnterBackground`... Not a blocker but wanted to mention it.